src/fmu/dataio/dataio.py

```diff
@@ -78,24 +78,40 @@ def _validate_variable(key, value, legals) -> bool:
     return True
 
 
-def _check_global_config(globalconfig: dict, strict: bool = True):
+def _check_global_config(
+    globalconfig: dict, strict: bool = True, action: str = "error"
+) -> bool:
     """A check/validation of static global_config.
 
-    Currently not a full validation. For now, just check that some required
-    keys are present in the config and raise if not.
+    Currently far from a full validation. For now, just check that some required
+    keys are present in the config and warn/raise if not.
     """
 
     if not globalconfig and not strict:
-        warn(
-            "Empty global config, expect input from environment_variable instead",
-            UserWarning,
+        logger.info(
+            "Empty global config, expect input from environment_variable insteac"
         )
-        return
+        return False
 
     config_required_keys = ["access", "masterdata", "model"]
```

Comment:
Have not traced it all the way through, but this this relate to the other warning about missing `masterdata` or is it redundant? https://github.com/equinor/fmu-dataio/pull/269/commits/19857a43b519c59e5a466c0fb4fdd9e3f0b3bcad#r1020934011

Reply:
I change it a bit; also so it triggers a `PendingDeprecationWarning` instead of a `UserWarning`

The keys in this function allow for a first pass of a config; if not from a file than an ENV variable is checked; finally it is checked with `strict=True` and `action="warn"` for` ExportData()`, wihile `action = "error"` for `InitializeCase()`

Better now?

Reply:
Probably fine, I was just wondering if we are checking for the same thing twice, and that the first was perhaps redundant.